pkg/visitors/remove.go

```diff
@@ -5,14 +5,18 @@
 package visitors
 
 import (
+	"fmt"
+	"io/ioutil"
+
 	"github.com/linuxboot/fiano/pkg/uefi"
 )
 
 // Remove all firmware files with the given GUID.
 type Remove struct {
 	// Input
-	Predicate func(f uefi.Firmware) bool
-	Pad       bool
+	Predicate  func(f uefi.Firmware) bool
+	Pad        bool
+	RemoveDxes bool // I hate this, but there's no good way to work around our current structure
```

Comment:
I still prefer how I did it in futk but you know that already.

The whole blacklist file is a poor fit with the rest of the nice package.